### PR TITLE
feat(CortexDuo): add presence

### PR DESCRIPTION
**Pull request by Roxza**
## Description 
A service that lets you talk to AI and others anonymously.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Ekran Resmi 2022-08-27 23 44 30
Ekran Resmi 2022-08-27 23 46 39

</details>

### CODE REVIEW

**Review by RisingSunLight42 — CHANGES_REQUESTED** [review]
Remove duplicate version in metadata, resolve checks, remove the js file and make a better presence. The current one is really poor.

File: websites/C/Cortex Duo/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+		"id": "852798440680456212"
+	},
+	"service": "Cortex Duo",
+	"description": {
+		"tr": "Her gün konuşarak, sosyalleşebileceğin özel bir platform.",
+		"en": "A special platform where you can socialize by talking every day."
+	},
+	"version": "1.0.0",
+	"url": "cortexduo.com",
+	"version": "25.08.22",
```
Comment (DarkVillager) [inline websites/C/Cortex Duo/dist/metadata.json:14]: what

File: websites/C/Cortex Duo/dist/presence.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    };
+    if (window.location.pathname.toLowerCase() === "/") {
+        presenceData.details = "CortexDuo'da takılıyor...";
+        presenceData.state = "cortexduo.com";
+    }
+    if (presenceData.details)
+        presence.setActivity(presenceData);
+    else
+        presence.setActivity();
+});
```
Comment (DarkVillager) [inline websites/C/Cortex Duo/dist/presence.js:16]: Dont include this

File: websites/C/Cortex Duo/presence.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+		}
+
+	if (window.location.pathname.toLowerCase() === "/") {
+		presenceData.details = "CortexDuo'da takılıyor...";
+		presenceData.state = "cortexduo.com";
+	}
```
Comment (DarkVillager) [inline websites/C/Cortex Duo/presence.ts:13]: More pages otherwise the presence is too low quality.

**Review by DarkVillager — CHANGES_REQUESTED** [review]

**Comment by DarkVillager** [comment]
@Roxza Maybe acutally use visual studio code so you can format the code?

File: websites/C/Cortex Duo/presence.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+}),
+browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo",
+    startTimestamp: browsingTimestamp,
+  };
+
+  if (window.location.pathname.toLowerCase() === "/") {
```
Comment (DarkVillager) [inline websites/C/Cortex Duo/presence.ts:12]:
```suggestion
  if (pathname === "/") {
```

File: websites/C/Cortex Duo/presence.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    if (document.querySelector<HTMLElement>("#duo_loading").style.display === "flex") {
+      presenceData.details = "CortexDuo'da eşleşiyor...";
+      presenceData.state = "cortexduo.com";
```
Comment (DarkVillager) [inline websites/C/Cortex Duo/presence.ts:15]: If you want them to go the site just add a button.
```suggestion
```

File: websites/C/Cortex Duo/presence.ts [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    } else if (!document.querySelector<HTMLElement>('button[class="search_humans"]')) {
+      presenceData.details = "Sohbet ediyor...";
+      presenceData.state = "cortexduo.com";
```
Comment (DarkVillager) [inline websites/C/Cortex Duo/presence.ts:18]: Same as above
```suggestion
```

File: websites/C/Cortex Duo/presence.ts [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+    } else if (document.querySelectorAll<HTMLElement>("#line_chat").length > 5) {
+      presenceData.details = "Yapay Zeka ile sohbet ediyor...";
+      presenceData.state = "cortexduo.com";
```
Comment (DarkVillager) [inline websites/C/Cortex Duo/presence.ts:21]: Same as above
```suggestion
```

File: websites/C/Cortex Duo/presence.ts [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+    } else {
+      presenceData.details = "CortexDuo'da takılıyor...";
+      presenceData.state = "cortexduo.com";
```
Comment (DarkVillager) [inline websites/C/Cortex Duo/presence.ts:24]: Same as above
```suggestion
```

File: websites/C/Cortex Duo/presence.ts [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+    }
+  } else if (window.location.pathname.toLowerCase() === "/error") {
+    presenceData.details = "Hata sayfasında.";
+    presenceData.state = "cortexduo.com";
```
Comment (DarkVillager) [inline websites/C/Cortex Duo/presence.ts:28]: Same as above
```suggestion
```

File: websites/C/Cortex Duo/presence.ts [hunk 10]
```diff
@@ -0,0 +1,32 @@
+const presence = new Presence({
+    clientId: "1011560036058800198",
+}),
+browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo",
```
Comment (DarkVillager) [inline websites/C/Cortex Duo/presence.ts:8]:
```suggestion
    largeImageKey: "https://i.imgur.com/iTS3Dcq.png",
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

**Comment by EncryptedDev** [comment]
Mark conversations as resolved as you've completed them

**Comment by EncryptedDev** [comment]
Hello 👋🏼 

Unfortunately, I believe this PR is low quality as it doesn't offer many features/functions on the presence itself. You could think about adding dynamic features and a more hollistic website-compatability

**Pull request closed without merging**